packages/roosterjs-content-model-plugins/lib/autoFormat/numbers/transformOrdinals.ts

```diff
@@ -72,7 +72,8 @@ function getNumericValue(text: string, checkFullText: boolean = false): number |
     const number = checkFullText ? text : text.substring(0, text.length - ORDINAL_LENGTH);
     const isNumber = /^-?\d+$/.test(number);
     if (isNumber) {
-        return parseInt(text);
+        const numericValue = parseInt(number);
+        return numericValue < 20 ? numericValue : parseInt(number.substring(number.length - 1));
```

Comment:
nit: Is it possible to be negative number? For example, -21 is less than 20, but it should be -21st (I think?)

Reply:
I fixed to use absolute values.

Reply:
But I think negatives are not used for ordinal numbers, but I'm not sure.